overlord/snapstate/backend_test.go

```diff
@@ -206,6 +206,14 @@ func (f *fakeSnappyBackend) RemoveSnapData(info *snap.Info) error {
 	return nil
 }
 
+func (f *fakeSnappyBackend) RemoveSnapSharedData(info *snap.Info) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "remove-snap-shared-data",
+		name: info.MountDir(),
```

Comment:
Yeah, let's not overthink this too much for now. It's just a way to print out what the backend had at hand when the given method was called.